src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import BooksList from './components/BooksList';
+import BooksForm from './components/BooksForm';
+import './App.css';
+
+const App = () => (
+  <div className="App">
+    <BooksList />
+    <BooksForm />
+  </div>
+);
+
+export default App;
```

Comment:
Please relocate this this file to `"src/components/"` folder. Has nothing to do with performance or workings of  the project, but this is how the guideline wants it structured. Hope you understand, thanks